.drone.yml

```diff
@@ -91,23 +155,55 @@ steps:
   commands:
   - ./.ci/travis/travis.sh javac16
 
-- name: no-error-test-sbe
-  image: maven:3.6.3-adoptopenjdk-8
-  commands:
-  - ./.ci/travis/travis.sh no-error-test-sbe
-
 ---
 kind: pipeline
-name: pipeline6
+name: assembly-site_sbe
+
+volumes:
+- name: m2-cache
+  temp: {}
 
 steps:
+- name: restore-cache
+  image: maven:3.6.3-adoptopenjdk-8
+  failure: ignore
+  volumes:
+  - name: m2-cache
+    path: /.m2
+  commands:
+  - ./.ci/drone-io.sh restore-maven-cache
+
 - name: jdk14-assembly-site
   image: maven:3.6.3-jdk-14
+  environment:
+    MAVEN_OPTS: "-Dmaven.repo.local=/.m2"
+  volumes:
+  - name: m2-cache
+    path: /.m2
   commands:
   - ./.ci/travis/travis.sh jdk14-assembly-site
 
 - name: assembly/site with OpenJDK11
   image: maven:3.6.3-adoptopenjdk-11
+  environment:
+    MAVEN_OPTS: "-Dmaven.repo.local=/.m2"
+  volumes:
+  - name: m2-cache
+    path: /.m2
   commands:
   - "mvn -e --no-transfer-progress package -Passembly && \
      mvn -e --no-transfer-progress site -Dlinkcheck.skip=true"
+
+- name: no-error-test-sbe
+  image: maven:3.6.3-adoptopenjdk-8
+  environment:
+    MAVEN_OPTS: "-Dmaven.repo.local=/.m2"
+  volumes:
+  - name: m2-cache
+    path: /.m2
```

Comment:
@sukolenvo I am curious, why do we need to replicate "volumes" for each step? The volumes at the end of the file isn't a global area and can't cover it?

Reply:
@rnveach , I am not sure if this answer your question  , but please read https://github.com/checkstyle/checkstyle/pull/10679#discussion_r699726182

Reply:
Volume at the end is my mistake, I suppose to move it at top of  pipilene, but looks like I did copy.

We need to remove block at the end of file

Reply:
https://github.com/checkstyle/checkstyle/pull/10780